cmake/fetch_reconverse/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# This directory is solely for fetching reconverse.
```

Comment:
Out of curiosity, does this file need its own directory, or could it be (e.g.) `cmake/fetch_reconverse.cmake`?